release/NativeEditPlugin/scripts/NativeEditBox.cs

```diff
@@ -230,17 +240,26 @@ private void onTextChange(string newText)
 			return;
 		
 		this.objUnityInput.text = newText;
-		if (this.objUnityInput.onValueChanged != null) this.objUnityInput.onValueChanged.Invoke(newText);
+		if (this.objUnityInput.onValueChanged != null)
+			this.objUnityInput.onValueChanged.Invoke(newText);
 	}
 
 	private void onTextEditEnd(string newText)
 	{
 		this.objUnityInput.text = newText;
-		if (this.objUnityInput.onEndEdit != null) this.objUnityInput.onEndEdit.Invoke(newText);
+		if (this.objUnityInput.onEndEdit != null)
+			this.objUnityInput.onEndEdit.Invoke(newText);
 	}
 
 	public override void OnPluginMsgDirect(JsonObject jsonMsg)
 	{
+		StartCoroutine(PluginsMessageRoutine(jsonMsg));
```

Comment:
Maybe we could run the coroutine on the PluginMsgHandler gameobject, I think that should always be active.